logdetective/server/server.py

```diff
@@ -85,10 +88,45 @@ async def lifespan(fapp: FastAPI):
     # Ensure that the database is initialized.
     logdetective.server.database.base.init()
 
+    # Set up a queue for LLM requests.
+    # When talking to an LLM service, there's a limited number of requests
+    # we can make per-minute.
+    fapp.llm_queue = asyncio.Queue(maxsize=SERVER_CONFIG.inference.max_queue_size)
+    asyncio.create_task(process_queue(fapp.llm_queue))
```

Comment:
wow, lifespan function doing some serious heavy lifting

Reply:
I agree, it might not be the correct place to initialize this. I'm just not really aware of a better one. This is the only place in the code with an obvious hook to do early initialization.

I'm not very familiar with FastAPI, though. If there's a better option, I'm all for it.